app/controllers/api/v1/documents_controller.rb

```diff
@@ -3,11 +3,13 @@
 module Api
   module V1
     class DocumentsController < ApiController
-      CONTENT_TYPE = 'multipart/form-data'
+      prepend_before_action :set_restricted_request_content_type, only: :create
 
       def create
         document = Document.create!(document_attributes)
         render json: document, status: 201
+      rescue ActiveSupport::MessageVerifier::InvalidSignature
+        Document.create!(file: nil)
```

Comment:
Sorry, I've just noticed this. How come we create an empty file if the signature's invalid?